tron/core/actionrun.py

```diff
@@ -672,7 +675,7 @@ def handle_action_command_state_change(self, action_command, event):
             return self.transition_and_notify('started')
 
         if event == ActionCommand.FAILSTART:
-            return self._exit_unsuccessful(None)
+            return self._exit_unsuccessful(-2)
```

Comment:
Out of curiosity, why -2? Any non-zero integer works, right?

Reply:
@qui nit: could we define a named constant then?  -2 seems a bit arcane

Reply:
Yeah, that was done here: https://github.com/Yelp/Tron/pull/578
I think that was after I started this PR, I'll update it in a follow-up - there's a bug elsewhere I need to fix anyway.